modules/navigation/nav_utils.h

```diff
@@ -318,6 +324,18 @@ struct PerformanceData {
 	int pm_edge_connection_count = 0;
 	int pm_edge_free_count = 0;
 	int pm_obstacle_count = 0;
+
+	void reset() {
+		pm_region_count = 0;
```

Comment:
can't you memset it with sizeof(PerformanceData). If you add some fields you won't have to update here

Reply:
Not sure if this struct will always be just some simple int values so I rather be more explicit about what reset is doing.